Users.js

```diff
@@ -63,6 +64,13 @@ class Users extends React.Component {
   };
 
   static manifest = Object.freeze({
+    query: {
+      initialValue: {
+        search: '',
+        filters: 'active.Active',
+        sort: 'Name',
+      },
+    },
```

Comment:
I'm not too happy about defining this statically. In the old version, the initial URL within the module was defined in the package file: and I see that in your revised stripes-core, that is still the case. Can't we make it that when you first click on the Users link and stripes-core send you to this application (specifically, to `/users/?filters=active.Active&sort=Name`) the value of the `query` resource gets set from that URL? In fact, wouldn't that happen _on its own_, in light of the code as you've written it?

Reply:
See comment below...